app.js

```diff
@@ -2,20 +2,34 @@ const express = require('express')
 const app = express()
 const endpoints = require('./endpoints.json')
 
-const { getTopics } = require('./controller/get.controller')
+const { getTopics, getArticleById } = require('./controller/get.controller')
 
 app.get('/api/topics', getTopics)
 
 app.get('/api', (req, res) => {
     res.status(200).send({endpoints})
 })
 
+app.get('/api/articles/:article_id', getArticleById)
+
+
 
 //Invalid route handling
 app.all('*', (req, res) => {
     res.status(404).send({msg: '404 Error: Path not found'})
 })
 
+//Custom error handling
+app.use((err, req, res, next) => {
+    if (err.status && err.msg) {
```

Comment:
It would in my opinion be better below to use err.status rather than hardcode the 404 status